runtime/include/chpltypes.h

```diff
@@ -191,6 +191,13 @@ typedef int64_t chpl_bool64;
 typedef void (*chpl_fn_p)(void*); // function pointer for runtime ftable
 typedef int16_t chpl_fn_int_t;    // int type for ftable indexing
 
+// Function table names and information, for VisualDebug use
+typedef struct _chpl_fn_info {
+  const char *name;
+  int fileno;
+  int lineno;
+} chpl_fn_info;
+
```

Comment:
I added this type here since chpl_fn_p was defined here also.    I'd add it to the visual debug header if I could generate this table only if VisualDebug.chpl module was used.   I don't know how to make that check.   Currently, this needs to be there for every chpl program, whether they use VisualDebug.chp or not.    This type is directly related to the chpl_fn_p type.